viseron/domains/face_recognition/__init__.py

```diff
@@ -115,13 +123,13 @@ class AbstractFaceRecognition(AbstractPostProcessor):
     def __init__(self, vis, component, config, camera_identifier) -> None:
         super().__init__(vis, config, camera_identifier)
         self._faces: dict[str, FaceDict] = {}
-
-        for face_dir in os.listdir(config[CONFIG_FACE_RECOGNITION_PATH]):
-            if face_dir == "unknown":
-                continue
-            vis.add_entity(
-                component, FaceDetectionBinarySensor(vis, self._camera, face_dir)
-            )
+        if not config[CONFIG_USE_SUBJECTS]:
```

Comment:
Need to find a way to move this to the `compreface` code instead of the base class.
Let me know if you need help with that!

Reply:
The reason for this and the part below is because the base class is where entities get created based on the file structure in the face_recognition path This feature stops this from occurring, so the only alternative would be to remove it completely from the base class and implement it in each implementation of it. This was not only less modifications, but seemed more elegant. With the below part, any none-compreface plugin will default to false and implement as usual. The only downside is that a user could turn this option to true for other plugins, and have no entities created.
What do you suggest in order to prevent adding entities at the base class level in this case?

Reply:
I just thought of a way to possibly do this while trying to go to bed, but haven't referenced the code to confirm if it's a good way of doing this. 

What if I put an optional boolean parameter that defaults to true in this init method which prevents the entity initialization when false. Then in the compreface implementation pass the value of use_subjects? 

You'll end up seeing this before I get a chance to implement this tomorrow, so let me know if this makes sense.

Reply:
Sounds like a good solution to me!

Reply:
3rd commit addresses this.